fish-rust/src/builtins/type.rs

```diff
@@ -103,9 +103,9 @@ pub fn r#type(parser: &Parser, streams: &mut IoStreams, argv: &mut [&wstr]) -> O
                     if props.is_copy() {
                         let path = props.copy_definition_file().unwrap_or(L!(""));
                         if path.is_empty() {
-                            comment.push_utfstr(&wgettext_fmt!(", copied interactively"));
+                            comment.push_utfstr(&wgettext!(", copied interactively"));
                         } else if path == L!("-") {
-                            comment.push_utfstr(&wgettext_fmt!(", copied via `source`"));
+                            comment.push_utfstr(&wgettext!(", copied via `source`"));
```

Comment:
good improvement for our current situation.
In future we should move to `std::fmt` syntax for which there are off-the-shelf checks that can run at compile time.